packages/collector/test/tracing/database/ioredis/test.js

```diff
@@ -42,8 +43,56 @@ function checkConnection(span, setupType) {
 //    export AZURE_REDIS_CLUSTER=team-nodejs-redis-cluster-tekton.redis.cache.windows.net:6380
 //    export AZURE_REDIS_CLUSTER_PWD=
 ['default', 'cluster'].forEach(setupType => {
+  if (setupType !== 'cluster') {
+    mochaSuiteFn('When allowRootExitSpan: true is set', function () {
+      this.timeout(config.getTestTimeout() * 4);
+
+      globalAgent.setUpCleanUpHooks();
+      const agentControls = globalAgent.instance;
+      let controls;
+
+      before(async () => {
+        controls = new ProcessControls({
+          useGlobalAgent: true,
+          appPath: path.join(__dirname, 'allowRootExitSpanApp'),
+          env: {
+            REDIS_CLUSTER: setupType === 'cluster'
```

Comment:
🤔

Reply:
Can be removed. @abhilash-sivan Please just set to false.